pull.py

```diff
@@ -57,8 +57,40 @@
         'FDLv1.2',
         'FDLv1.3',
     ],
+    'FreeArt': [ # any version
+        'LAL-1.2',
+        'LAL-1.3',
+    ],
     'FreeBSDDL': ['FreeBSD'],  # unify (multi-tag)
-    # FIXME: still working through this
+    'NPL': [ #any version
+        'NPL-1.0',
+        'NPL-1.1',
+    ],
+    'OSL': [ # any version through 3.0
+        'OSL-1.0',
+        'OSL-1.1',
+        'OSL-2.0',
+        'OSL-2.1',
+        'OSL-3.0',
+    ],
+    'RPL': [ # any version - Note that FSF website does not state any version, but references version 1.3 in the URL.  It is assumed that it also covers version 1.1 and 1.5, but this should be verified with FSF.
+        'RPL-1.1',
+        'RPL-1.3',
+        'RPL-1.5',
+    ],
+    'Unicode': [ # any version
+        'Unicode-DFS-2015',
+        'Unicode-DFS-2016',
+    ],
+    'W3C': [ # any version
+        'W3C',
+        'W3C-20150513',
+        'W3C-19980720',
+    ],
+    'Zope2.0': [ # Versions 2.0 and later
```

Comment:
The FSF wording for this isn't “Versions 2.0 and later”, it's “[versions 2.0 and 2.1][1]”.

[1]: https://www.gnu.org/licenses/license-list.html#Zope20